src/js/controllers/messageController.js

```diff
@@ -136,4 +142,19 @@ export const messageScreen = (state) => {
         }
         
     });
+
+    const searchResUI = document.querySelector('.search-results');
+
+    document.querySelector('.search-btn').addEventListener('click', async e =>  {
+        e.preventDefault();
+        const btn = e.target.closest('.search-btn');
+        console.log(btn);
```

Comment:
not a big thing, but i think logs can be deleted when commiting just so the master branch is clean

Reply:
We can create an issue to clean logs

Reply:
Added to issue https://github.com/Yuxuan-Lin/VioLegacy/issues/24